project/AppDependencies.scala

```diff
@@ -22,17 +22,17 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.20.0",
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.24.0",
     "uk.gov.hmrc"                  %% "simple-reactivemongo"      % "8.0.0-play-28",
```

Comment:
Can you please migrate this to hmrc-mongo https://github.com/hmrc/hmrc-mongo/blob/main/MIGRATION.md